### PR TITLE
Bug 1043953 - WebIDE should be able to build'n push gaia apps +autoland

### PR DESCRIPTION
**Pull request by ochameau**
https://bugzilla.mozilla.org/show_bug.cgi?id=1043953

### CODE REVIEW

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=472286fe224d)

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dfb44532bc36)

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8fffd3430ce4)

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a3cb841eead2)

File: build/webapp-zip.js [hunk 1]
```diff
@@ -94,6 +94,9 @@ WebappZip.prototype.isExcludedFromZip = function(file) {
     function isReadme(file) {
       return /README/.test(file.path);
     },
+    function isPackageJSON(file) {
+      return /package.json/.test(file.path);
```
Comment (RickyChien) [inline build/webapp-zip.js]: It should be /package\.json/

**Comment by try-server-hook** [comment]
[ochameau null (ochameau)](https://github.com/ochameau) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f623569b2ea0)

**Pull request merged**